src/main/resources/messages.properties

```diff
@@ -34,14 +34,16 @@ public.user.activation.heading.text=Create a password
 public.user.activation.title.text=User Activation
 public.user.activation.password.one=Create your password
 public.user.activation.password.two=Repeat password 
-public.user.activation.expired.heading=Sorry, your link has expired
 public.user.activation.expired.title=Link Expired
-public.user.activation.expired.text=Please contact customer services for more information.
-public.user.activation.expired.link.click.here=Click here 
-public.user.activation.expired.link.sing.into=to sign into the service or register your details
+public.user.activation.expired.text.p1.1=To get a new link, you need to
+public.user.activation.expired.text.p1.2=create your account again
+public.user.activation.expired.text.p2=You can use the same email and password.
```

Comment:
not a biggie but I don't think this naming conventions is following the standard

Reply:
Yeah, I don't understand p1 and p2. Are they "part1" and "part2"?